grammars/php.cson

```diff
@@ -1519,25 +1524,25 @@
             'name': 'punctuation.definition.character-class.php'
         'end': '\\]'
         'name': 'string.regexp.character-class.php'
-        'patterns': [
-          {
-            'include': '#single_quote_regex_escape'
```

Comment:
I commented out the redundant sections after I placed the single_quote_regex_escape content into the patterns array as its own item, rather than being a referenced sub-item. I wasn't sure whether I should completely remove the comments or not prior to review and finalization. The code works, just the scoping was off. Does that explain it better?